sources/packages/api/src/services/bceid/account-details.model.ts

```diff
@@ -0,0 +1,17 @@
+export interface AccountDetails {
+  user: UserDetails;
+  institution: InstitutionDetails;
+}
+
+export interface UserDetails {
+  guid: string;
+  displayName: string;
+  firstname: string;
+  surname: string;
+  email: string;
+}
+
+export interface InstitutionDetails {
+  guid: string;
+  legalName: string;
+}
```

Comment:
Confused between this and bceid-account.model.ts. Lets talk about it

Reply:
Renamed to dto.